frodo.js

```diff
@@ -5,41 +5,42 @@
  * for the extras/ plugins if they are loaded into a presentation.
```

Comment:
SCRIPT-This hardcoded value might be better suited as a constant at the top of the file.